src/main/io/vtx_settings_config.c

```diff
@@ -35,43 +35,86 @@ PG_RESET_TEMPLATE(vtxSettingsConfig_t, vtxSettingsConfig,
     .freq = 0
 );
 
-//Saves the given band/channel values to configuration settings.
-// band:  Band value (1 to 5).
-// channel:  Channel value (1 to 8).
-void vtxSettingsSaveBandAndChannel(uint8_t band, uint8_t channel)
+static bool vtxChangePending = false;
+
+static void vtxChangeSetting(bool changed)
 {
-    bool modFlag = false;
-    if (band != vtxSettingsConfigMutable()->band) {
+    if (changed && !vtxChangePending) {
+        vtxChangePending = changed;
```

Comment:
Setting this to true here will make it easier to read.

Reply:
Done.